docs/source/model_parameters.md

```diff
@@ -0,0 +1,294 @@
+# Simulation Model
+
+## Introduction
+
+Simulation model parameters describe properties of all relevant elements of the observatory including site,
+telescopes, and calibration devices.
+The management, definition, derivation, verification, and validation of the simulation model is central to the functionality of simtools.
+
+Examples for model parameters are the atmospheric models (profile, transmission) or geomagnetic field characteristics for the site,
+description of optical and mechanical properties of the telescope, and detector plan, trigger, and readout settings for the cameras.
+
+For simplicity, the term parameter is used for values, (multi-dimensional) vectors of values
+(e.g., the mirror reflectivity vs wavelength and photon incident angle), functions, and algorithms
+(e.g., the telescope trigger algorithm).
+Parameters might be fixed in time (e.g., the number of mirrors on a certain telescope), or variable
+on different time scales (e.g., compare the degradation of the mirror reflectivity over several months
+with the nightly variability of the atmospheric parameters).
+
+The major components to handle the simulation model in simtools are:
+
+1. **Model parameter databases:** The simulation model is stored in [mongoDB databases](databases.md#databases).
+The {ref}`db_handler module <DBHANDLER>` provides reading and writing interfaces to the database.
```

Comment:
I would add to this that we have the developer tools to work with the DB. We rarely would access db_handler directly when working with the DB

Reply:
I would also add that the model parameters (in the DB) unambiguously define the array, as it contains all the information necessary for the simulations.

Reply:
The introduction is already too long! Don't want to add something.

- on db_handler: there is a complete section further down describing this. I think in the intro it is enough that we have tools to read and write to the db
- on the definition of the array: isnt this the first sentence of the intro, where we say 'all relevant parameters of the observation are defined'?